src/Init/Data/BitVec/Bitblast.lean

```diff
@@ -346,6 +346,10 @@ theorem getMsbD_sub {i : Nat} {i_lt : i < w} {x y : BitVec w} :
   · rfl
   · omega
 
+theorem getElem_sub {i : Nat} {x y : BitVec w} (h : i < w) :
+    (x - y)[i] = (x[i] ^^ ((~~~y + 1#w)[i] ^^ carry i x (~~~y + 1#w) false)) := by
+  rw [sub_toAdd, neg_eq_not_add, getElem_add]
```

Comment:
Not a big deal since the proof is so short, but I think it's a good principle to reuse existing proofs:
```suggestion
    simp [← getLsbD_eq_getElem, getLsbD_sub, h]
```

Reply:
Oh! Indeed exactly like you did below. :-)